pkg/env/web_env.go

```diff
@@ -96,10 +98,20 @@ func fetchEnvHTTP(envKey string, u *url.URL) (string, error) {
 		return "", err
 	}
 
-	if username != "" && password != "" {
-		req.SetBasicAuth(username, password)
+	claims := &jwt.StandardClaims{
+		ExpiresAt: int64(15 * time.Minute),
+		Issuer:    username,
+		Subject:   envKey,
+	}
+
+	token := jwt.NewWithClaims(jwt.SigningMethodHS512, claims)
+	ss, err := token.SignedString([]byte(password))
```

Comment:
Operator will need to have read access to the password and it needs to be per minio instance but that is not relevant here

Reply:
This secret is actually operator webhook secret not the tenant secret, so even if lost or compromised can be regenerated.